test/jit/test_tracer.py

```diff
@@ -2074,8 +2114,5 @@ def forward(self, input_map: Dict[str, List[Tensor]]) -> Tensor:
         input_map = {"1" : [torch.rand(2, 2), torch.rand(2, 2)], "3" : [torch.rand(2, 2), torch.rand(2, 2)]}
         model = testA()
         traced_model = torch.jit.trace(model, input_map)
-        # the dict should not be inlined as constants in the IR
-        self.assertFalse("CONSTANTS" in str(traced_model.code))
```

Comment:
we should keep this check? to make sure dict tensor values are not inlined as `CONSTANTS` (which is a faulty behavior before https://github.com/pytorch/pytorch/pull/39601).

Reply:
I think we shouldn't check `CONSTANTS` directly (the fact that tensors aren't inlined is an implementation detail), we should just pass different tensors as value and make sure the output actually changes. The tests I added do that.